addon/utils/text.js

```diff
@@ -89,9 +91,8 @@ export function trimLongDataInElement (element) {
   if (textWidth > width) {
     text = trimDataToFit(text, width)
     tooltip = $(element).data('text')
+    $element
```

Comment:
The new test does fail in Ember 2.3, but if you run the demo app under Ember 2.3, you will see that the `frost-select` does not really work at all, even if you completely disable the trim feature. 

As such, I am not sure how to proceed. Do we need to continue Ember 2.3 support? The test passes in Ember 2.4.